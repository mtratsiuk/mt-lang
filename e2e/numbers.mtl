(print 1)
(print 1_000)
(print -1_000)
(print 1.2)
(print .2)
(print 1.)
(print 1_000_000.2_222)
