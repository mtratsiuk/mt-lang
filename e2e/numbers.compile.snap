const __mtl_curry = f => (...args) => {
  if (args.length >= f.length) {
    return f(...args);
  }

  return __mtl_curry(f.bind(null, ...args));
};

console.log(String(1));

console.log(String(1000));

console.log(String(-1000));

console.log(String(1.2));

console.log(String(0.2));

console.log(String(1));

console.log(String(1000000.2222));
